pymodbus/client/serial.py

```diff
@@ -210,6 +216,7 @@ def connect(self):
                 stopbits=self.comm_params.stopbits,
                 baudrate=self.comm_params.baudrate,
                 parity=self.comm_params.parity,
+                exclusive=self.comm_params.exclusive,
```

Comment:
Isn't this the part of the sync client where I'm already passing the exclusive param to pyserial? Or are you referring to something different?